src/actions/post_build.rs

```diff
@@ -281,11 +283,17 @@ fn parse_diagnostics(message: &str, group: u64) -> Option<FileDiagnostic> {
                 secondary_message.push_str(&format!("\n\n{}", secondary_label));
             }
         }
+        let severity = Some(if secondary_span.is_primary {
+            DiagnosticSeverity::Warning
```

Comment:
Should the severity come from the message (i.e., `message.level`) rather than always being `Warning`? I would think that is this was an error message with multiple error spans, all primary spans should have the error severity.

Reply:
Yep good spot, will fix